.travis.yml

```diff
@@ -35,6 +35,7 @@ install:
   - pip install unidecode
   - pip install pycountry
   - pip install suds-jurko==0.6
+  - pip install seur
```

Comment:
have you tried to install `genshi` before `seur` ?

pip install genshi
pip install seur